scripts/dcos-account.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh 
+#
+# Write DC/OS credentials to file and update environment
+# variables accordingly.
+#
+
+# check if the SSL file exists and if not assume it needs
+# the MESOS_SANDBOX prefix.
+write_cred_file()
+{
+    if [ -n "$DCOS_SERVICE_ACCOUNT_CREDENTIAL" ] && [ "$DCOS_SERVICE_ACCOUNT_CREDENTIAL" != file* ]; then
```

Comment:
The bash equivalent of `startsWith`. This is checking whether the left side of the check starts with "file".